packages/renderer/src/lib/image/PullImage.svelte

```diff
@@ -169,6 +172,21 @@ onMount(() => {
   selectedProviderConnection ??= providerConnections.length > 0 ? providerConnections[0] : undefined;
 });
 
+onMount(async () => {
+  const configuration = await window.getConfigurationValue<string>('registries.preferredRegistries');
```

Comment:
```suggestion
  const configuration = await window.getConfigurationValue<string>('registries.preferred');
```

might be better to move the config values in the packages/api to avoid hardcoding strings here is an example

https://github.com/podman-desktop/podman-desktop/blob/84f1ded7044f64fd0fd4e38e89e68b03dc9e0bb7/packages/api/src/docker-compatibility-info.ts#L19-L22

Reply:
Added to packages/api